app/bundles/SmsBundle/Tests/Sms/TransportChainTest.php

```diff
@@ -49,11 +49,11 @@ protected function setUp()
         parent::setUp();
 
         $this->transportChain = new TransportChain(
-            'mautic.test.twilio.mock', $this->container->get('mautic.helper.integration'), $this->container->get('logger')
+            'mautic.test.twilio.mock',
+            $this->container->get('mautic.helper.integration')
         );
 
-        $this->twilioTransport = $this->getMockBuilder(TwilioApi::class)
-                                      ->disableOriginalConstructor()->getMock();
+        $this->twilioTransport = $this->createMock(TwilioApi::class);
 
         $this->twilioTransport
             ->method('sendSMS')
```

Comment:
`->willReturn('lol');`
TY!

Reply:
It does the same, just different syntax. Again, let's focus on the changes in this PR. Not the code around it.